Propulsion/Oarhole.md

```diff
@@ -0,0 +1,28 @@
+**_oarhole, oarport_** (English); _årehul_ (Danish); _Riemenloch_ (German)
+
+_**hár** f., pl. háir_ (Old Norse) [citation: [prose](https://onp.ku.dk/onp/onp.php?o31447)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=31982)]   
+_**há-bora** f., pl. háborurnar, haborum_ (Old Norse) [citation: [prose](https://onp.ku.dk/onp/onp.php?o30226)]  
+_**hamla** f., pl. hǫmlur_ (Old Norse) [citation: [prose](https://onp.ku.dk/onp/onp.php?o31115)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=31626)]   
+
+
+  Holes situated within the upper strakes (hull) of Viking ships, allowing the crew to man the oars. (Crumlin-Pedersen, 116)
+<div align="center">
+  
+  ![oarhole-lock from Hedeby ship](../images/oarports.png)  
+  Reconstruction drawing of a section of the upper strakes and oarports of a vessel excavated in Hafenstrasse, Schelswig-Holstein, Germany. (Crumlin-Pedersen, 1996, Figure 5.12, p116)
+
+</div>
+
+  A common practice was to levy taxes based on the amount of oarports present on a ship; Harthacnut's tax of 8 marks on each crew memeber, for example, was tabulated by the number oarsmen's places (oarports). (Jesch, 156) 
```

Comment:
Some brief discussion here of the difference between the 3 terms? One sentence, perhaps at the beginning, that summarizes Jesch's discussion?